src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,33 @@
+package codeu.chat.common;
+
+import java.io.IOException;
+import codeu.chat.util.Uuid;
+import codeu.chat.util.Time;
+
+public final class ServerInfo {
+	private final static String SERVER_VERSION = "1.0.0";
+	public Uuid version;
+  public final Time startTime;
+
+	public ServerInfo(){
```

Comment:
You've got two different no-argument `ServerInfo` constructors here, which isn't going to compile.

I'm afraid this is a semi-evil semi-deliberate issue with the first two small features: @mlk209 and @lysanderp41 were instructed to construct two different `ServerInfo` classes.

I think that a correct merge here would keep a no-argument constructor that uses the same logic as both off these no-argument constructors put together. For the constructors that take arguments, I'd have to look at where they are called: Do we need one constructor that takes both parameters? Do we need two constructors that each take one of the two parameters (and use the default value for the other)? Or, do we need both (so, a no-arg constructor, two one-arg constructors, and one two-arg constructor)?